app/assets/javascripts/discourse/app/routes/tags-intersection.js

```diff
@@ -1,9 +1,20 @@
 import TagShowRoute from "discourse/routes/tag-show";
+import { queryParams } from "discourse/controllers/discovery-sortable";
 
-export default TagShowRoute.extend({});
-
-// The tags-intersection route is exactly the same as the tags-show route, but the wildcard at the
+// The tags-intersection route is the same as the tags-show route, but the wildcard at the
 // end of the route (*additional_tags) will cause a match when query parameters are present,
 // breaking all other tags-show routes. Ember thinks the query params are addition tags and should
 // be handled by the intersection logic. Defining tags-intersection as something separate avoids
 // that confusion.
+export default TagShowRoute.extend({
+  controllerName: "tags.intersection",
+
+  init() {
+    this._super(...arguments);
+
+    // The only difference is support for `category` query param.
+    // Other routes include category in the route path.
+    this.set("queryParams", { ...queryParams });
```

Comment:
Would the following work?

```js
this.set("queryParams", queryParams);
```

I guess my question is: what does `{ ...queryParams }` do in this case?